multinet/multinet/attribute.py

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
Is this line blank for a specific reason?

Reply:
Nope, removed.